src/components/dicomViewer/DcmHeader/DcmHeader.scss

```diff
@@ -16,11 +16,12 @@ $headerBlue: #73bcf7;
     margin: 0.3rem;
     svg {
       color: #fff;
+      font-size: 0.6rem;
     }
   }
   .dicom-header__body-tools,
   .dicom-header__body-resets {
-    padding-right: 4rem;
+    padding-right: 2rem;
```

Comment:
Hard-coded centering is not responsive

Reply:
@jennydaman i have made adjustments to that 
please help me review

Reply:
Also, what is the least screen size we are working with. So i know how to adjust the other components.

Reply:
The minimum width ChRIS_ui is aiming to support is 992px throughout the app. Most of our current users are on laptops and wider screens.

Reply:
Okay 992px. I will work with that.